e2e/benchmarks/local-benchmark/index.html

```diff
@@ -269,7 +269,7 @@ <h2>TensorFlow.js Model Benchmark</h2>
         // compare results
         try {
           await showMsg(null);
-          expectObjectsClose(predictionData, referenceData);
+          expectObjectsClose(predictionData, referenceData, `${state.benchmark}-${state.architecture}`);
```

Comment:
It seems that it can catch `FaceLandmarkDetection-attention_mesh`. But how it can catch `bodypix` in `shouldCompareWithFuzzy` ?